src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -63,26 +65,19 @@ public async Task<int> Execute()
         {
             int exitCode;
 
-            this.logger.Verbosity = this.bootstrapSettings.LoggingVerbosity;
-            this.logger.ResumeOutput();
+            logger.Verbosity = bootstrapSettings.LoggingVerbosity;
+            logger.ResumeOutput();
 
-            var phase = this.bootstrapSettings.Phase;
+            var phase = bootstrapSettings.Phase;
             LogProcessingStarted(phase);
 
             try
             {
-                if (phase == AnalysisPhase.PreProcessing)
-                {
-                    exitCode = await PreProcess();
-                }
-                else
-                {
-                    exitCode = PostProcess();
-                }
+                exitCode = phase == AnalysisPhase.PreProcessing ? await PreProcess() : PostProcess();
```

Comment:
could you put the `?` and `:` on two separate lines? 

because `await` is only on one branch, and when reading w/o paying attention one might think that it's on both commands